database/createListingData.js

```diff
@@ -1,16 +1,23 @@
 const fs = require('fs');
-const file = fs.createWriteStream('./listingSeedData.csv');
+//for artillery testing
+const file = fs.createWriteStream('./postListings.csv');
+
+//seed Db for listings table
+// const file = fs.createWriteStream('./listingSeedData.csv');
 const loremIpsum = require('lorem-ipsum').loremIpsum;
 
 const listingTypeOptions = ['Entire place', 'Private room', 'Hotel room', 'Shared room'];
 const listingCategoryOptions = ['apartment', 'house', 'hotel', 'cabin', 'bnb'];
 const listingColumnNames = 'listing_id, listing_type, listing_category, night_price, avg_review, num_review, number_beds, listing_title, is_fav';
-
 file.write(listingColumnNames);
 
-// seedDB for listings table
-for (let i = 0; i < 1e7; i++) {
-  let listingId = i + 10001;
+  // for artillery testing in postListings.csv
+for (let i = 0; i < 40000; i++) {
+    let listingId = 1e7 - 50000;
+
```

Comment:
This file looks great, however I don't understand this for loop intention for "artillery testing in postListings.csv"

Reply:
Oh what that is for is that I had to insert a bunch of data into my database for the post or update artillery test, so I created 40000 high number listings in csv format and exported them to the postListings.csv file.    I then used the csv file in my artillery yml update file.